datalake/src/main/java/com/sequenceiq/datalake/service/sdx/VerticalScaleService.java

```diff
@@ -112,4 +115,22 @@ public StackV4Response waitCloudbreakClusterVerticalScale(Long id, PollingConfig
                 () -> stackV4Endpoint.get(0L, sdxCluster.getClusterName(), Collections.emptySet(), sdxCluster.getAccountId()));
     }
 
+    public FlowIdentifier updateDisksDatalake(SdxCluster sdxCluster, DiskUpdateRequest updateRequest, String userCrn) {
+        MDCBuilder.buildMdcContext(sdxCluster);
+        LOGGER.info("Environment Vertical Scale flow triggered for environment {}", sdxCluster.getName());
+        DatalakeDiskUpdateEvent datalakeDiskUpdateTriggerEvent = DatalakeDiskUpdateEvent.builder()
```

Comment:
same here

Reply:
no. this is the same way multiple events are being generated in the previous codes that were written. I am not changing it because it is just used once to create that specific event.